test/relations.test.js

```diff
@@ -832,8 +835,16 @@ describe('relations', function() {
               if (err) return done(err);
               should.exist(ch);
               ch.should.have.lengthOf(2);
-              var resultIdArr = [ch[0].id, ch[1].id];
-              assert.deepEqual(resultIdArr, idArr);
+              if (typeof idArr[0] === 'object') {
+                // mongodb returns `id` as an object
+                idArr[0] = idArr[0].toString();
+                idArr[1] = idArr[1].toString();
+                idArr.indexOf(ch[0].id.toString()).should.be.above(-1);
```

Comment:
same comment.. if we're checking index should not be -1, perhaps better to use `should.not.equal(-1)`.